geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -22,11 +27,12 @@ import org.locationtech.geomesa.index.utils.Explainer
 import org.locationtech.geomesa.utils.collection.{CloseableIterator, SelfClosingIterator}
 import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
 
-trait HBaseQueryPlan extends HBaseQueryPlanType {
+
+sealed trait HBaseQueryPlan extends HBaseQueryPlanType {
   def filter: HBaseFilterStrategyType
   def table: TableName
   def ranges: Seq[Query]
-  // note: entriesToFeatures encapsulates ecql and transform
+  def remoteFilters: Seq[HFilter]
```

Comment:
let's remove this since it's potentially confusing - the scanplan already has the remote filters configured in the scan objects